NAMESPACE

```diff
@@ -7,7 +7,6 @@ export(abline)
 export(arrows)
 export(axis)
 export(bgCol)
-export(bgCol.default)
```

Comment:
I think I'm convinced it doesn't break anything,so no reason not to take out the other ones too, right?